tests/testthat/test-run-nmtran.R

```diff
@@ -0,0 +1,217 @@
+context("testing NM-TRAN")
+
+# define constants
+MODEL_DIR_BBI <- file.path(dirname(ABS_MODEL_DIR), "test-nmtran-models")
+NONMEM_DIR <- Sys.getenv("BBR_TESTS_NONMEM_DIR", "/opt/NONMEM")
+REQ_NONMEM_VERSIONS <- c("nm73gf", "nm74gf", "nm75", Sys.getenv("BBR_TESTS_NONMEM_VERSION"))
+REQ_NONMEM_VERSIONS <- unique(REQ_NONMEM_VERSIONS[REQ_NONMEM_VERSIONS != ""])
+
+# Don't assume NONMEM is available if not on Metworx.
+if (Sys.getenv("METWORX_VERSION") == "" || Sys.getenv("SKIP_BBI_TEST") == "true") {
+  skip("test-workflow-bbi only runs on Metworx because it needs NONMEM installed")
+}
+skip_long_tests("skipping long-running bbi workflow tests")
+
+
+# Skip if required NONMEM versions are not installed
+if(!all(REQ_NONMEM_VERSIONS %in% basename(fs::dir_ls(NONMEM_DIR)))){
+  req_nm_ver_txt <- paste(REQ_NONMEM_VERSIONS, collapse = ", ")
+  skip(glue("test-run-nmtran-bbi only requires the following NONMEM versions {req_nm_ver_txt}"))
```

Comment:
"only requires" reads odd to me in this context. Drop "only"?

Also, missing colon after "versions"?